orchagent/routeorch.h

```diff
@@ -206,6 +206,8 @@ class RouteOrch : public Orch, public Subject
     unsigned int getNhgCount() { return m_nextHopGroupCount; }
     unsigned int getMaxNhgCount() { return m_maxNextHopGroupCount; }
 
+    void updateDefRouteState(string ip, bool add=false);
```

Comment:
Is this necessary to be a public function?

Reply:
Moved to private